lib/core/src/lib/viewer/components/img-viewer.component.spec.ts

```diff
@@ -354,13 +354,15 @@ describe('Test Img viewer component ', () => {
             component.readOnly = false;
             component.isEditing = true;
 
-            spyOn(component, 'save');
+            spyOn(component.isSaving, 'emit');
+            spyOn(component, 'save').and.callThrough();
             fixture.detectChanges();
             const saveButtonElement = fixture.debugElement.query(By.css('#viewer-save-button'));
             saveButtonElement.triggerEventHandler('click', null);
             tick();
 
             expect(component.save).toHaveBeenCalled();
+            expect(component.isSaving.emit).toHaveBeenCalled();
```

Comment:
Can you also test if that emits true or false? Also can you add test for second case as you have two usages of that in component?